scene/resources/animation.h

```diff
@@ -502,6 +502,13 @@ class Animation : public Resource {
 	static Variant blend_variant(const Variant &a, const Variant &b, float c);
 	static Variant interpolate_variant(const Variant &a, const Variant &b, float c);
 
+	template <class T>
+	static Vector<T> add_packed_array(Vector<T> a, Vector<T> b);
+	template <class T>
+	static Vector<T> subtract_packed_array(Vector<T> a, Vector<T> b);
+	template <class T>
+	static Vector<T> interpolate_packed_array(Vector<T> a, Vector<T> b, float c);
+
```

Comment:
```
static Vector<T> blend_packed_array(Vector<T> a, Vector<T> b, float c);
```
Could we also add a `blend_packed_array()` for consistency?

Reply:
I could, although this PR is specifically for supporting tweening the Packed Arrays, which doesn't technically need the method.